### ISSUE TITLE
Negative predicted score

### ISSUE THREAD

**Issue by zhj1ein** [zwx8981/UNIQUE#4]
Hi, I simply train on single koniq-10k dataset with mos between 1 -5 , I got negative predicted score on some images. Do you think that is possible?

**Comment by zwx8981** [comment]
Did you train the model by regression or ranking？ It is normal for the latter.



发自我的iPhone


------------------ Original ------------------

**Comment by zhj1ein** [comment]
I train the model by ranking. How to interprate negative scores?

**Comment by zwx8981** [comment]
With no direct fitting to the MOS，the quality range of ranking-based training is learned by the optimization process itself. You do not need to care about the negative score as long as the monoticity is consistent with MOS&nbsp;



发自我的iPhone


------------------ Original ------------------

**Comment by zhj1ein** [comment]
@zwx8981  Thanks for your reply. But when you do testing, you may still get negative scores on unseen testing images. How to compare the quality of testing images with negative scores.

**Comment by zwx8981** [comment]
A higher score indicates a better quality，for example, 1 is better than -1, and -1 is better than -2



发自我的iPhone


------------------ Original ------------------

**Comment by zhj1ein** [comment]
@zwx8981  When testing on other datasets, no fine-tuning needed right?

**Comment by zwx8981** [comment]
@pencilzhang While our method can directly train the model on multiple datasets, you can still evaluate the model in a cross-database setting, in which no data of the test dataset is incorporated into the training set.

**Comment by zhj1ein** [comment]
@zwx8981  Thanks! when you did learn by regression on multiple datasets, did you use continuous ranking annotation, fidelity loss, hinge loss? I am wondering contributions of each component in your training strategy.

**Comment by zwx8981** [comment]
I use mse (l2 loss) loss for regression. Hinge loss is only used for uncertainty estimation，which is neither used in linear re-scaling nor binary labeling. Continuous ranking annotation cannot be used to learn by regression, where we use linearly re-scaled MOS instead.



发自我的iPhone


------------------ Original ------------------

**Comment by zhj1ein** [comment]
@zwx8981 Sorry I should indicate that for continuous ranking and newly introduced loss, are they beneficial to learn by regression on one single dataset?

**Comment by zwx8981** [comment]
@pencilzhang On one single dataset, I think the fidelity loss may not brings much (if any) benefit.

**Comment by zhj1ein** [comment]
@zwx8981  Thank you very much for your explanation. Expect the interpretation of negative quality values predicted by ranking network, I think I have already understood other things.:)
